src/libs/import.c

```diff
@@ -680,6 +680,15 @@ static void _lib_import_update_preview(GtkFileChooser *file_chooser, gpointer da
       GdkPixbuf *tmp;
       GdkPixbufLoader *loader = gdk_pixbuf_loader_new();
       if (!gdk_pixbuf_loader_write(loader, buffer, size, NULL)) goto cleanup;
+      // Calling gdk_pixbuf_loader_close forces the data to be parsed by the
+      // loader. We must do this before calling gdk_pixbuf_loader_get_pixbuf.
+      GError *error = NULL;
+      gdk_pixbuf_loader_close(loader, &error);
+      if(error)
```

Comment:
No need to pass error and then free it. Just use NULL as the 2nd argument to `gdk_pixbuf_loader_close()` and check its return value.